code/modules/research/techweb/all_nodes.dm

```diff
@@ -641,6 +641,8 @@
 		"thermomachine",
 		"w-recycler",
 		"welding_goggles",
+		"techfab_departmental_engineering", "techfab_departmental_science", "techfab_departmental_medical", "techfab_departmental_cargo", "techfab_departmental_security", "techfab_departmental_service", //Singulostation edit - Printable techfabs and protolathes
+		"protolathe_departmental_engineering", "protolathe_departmental_science", "protolathe_departmental_medical", "protolathe_departmental_cargo", "protolathe_departmental_security", "protolathe_departmental_service" //Singulostation edit - Printable techfabs and protolathes
```

Comment:
Please split this up into individual lines, with a trailing comma at the end, following upstream code style.